lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -30,7 +30,7 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
-const IGNORE_THRESHOLD_IN_BYTES = 2048;
+const IGNORE_THRESHOLD_IN_BYTES = 4096;
```

Comment:
This was the only image audit on 2KB, we actually [documented](https://web.dev/uses-responsive-images/?utm_source=lighthouse&utm_medium=cli) this value as 25KiB though I have no idea where that came from. Either way 4KiB seems more than reasonable and inline with other image byte efficiency audits.

Reply:
have you opened a pr to fix that?

Reply:
Nope

Reply:
https://github.com/GoogleChrome/web.dev/pull/3498